Assets/Scripts/Enemies/EnemyMovement.cs

```diff
@@ -18,6 +19,17 @@ void Start()
     public virtual void Update()
     {
         EnemyController.Update(gameObject, Time.deltaTime);
+
+        if (fading)
+        {
+            Color c = GetComponent<Renderer>().material.color;
+            GetComponent<Renderer>().material.color = new Color(c.r, c.g, c.b, c.a - 1.0F / 10);
```

Comment:
Consider moving the `1.0F / 10` to a const float at the top. This will allow us to easily tweak this without having to track it down in the file.